tests/functional/search/sphinx_test.php

```diff
@@ -20,8 +20,29 @@ class phpbb_functional_search_sphinx_test extends phpbb_functional_search_base
 {
 	protected $search_backend = 'phpbb\search\backend\fulltext_sphinx';
 
+	protected function create_search_index($backend = null)
+	{
+		parent::create_search_index($backend);
+		$this->purge_cache();
+
+		if (!$backend || $this->search_backend == $backend)
+		{
+			// After creating phpBB search index, build Sphinx index
+			exec('sudo -S service sphinxsearch stop', $output, $retval); // Attemtp to stop sphinxsearch service in case it's running
+			exec('sudo -S indexer --all', $output, $retval); // Run sphinxsearch indexer
+			exec('sudo -S service sphinxsearch start', $output, $retval); // Attempt to start sphinxsearch service again
+		}
+	}
+
 	public function test_search_backend()
 	{
-		$this->markTestIncomplete('Sphinx Tests are not supported');
+		if ($this->db->sql_layer != 'mysqli') // Sphinx test runs on MySQL/MariaDB only so far
```

Comment:
https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/search/backend/fulltext_sphinx.php#L203

Reply:
Generally yes, but this is with regards to the unit testing. The sample file is mysql only and the same applies for the tests on GitHub Actions. Therefore I would keep it like this for now. Created a ticket here: https://tracker.phpbb.com/browse/PHPBB3-16954